test/hotspot/jtreg/vmTestbase/nsk/share/jvmti/ArgumentHandler.java

```diff
@@ -147,8 +123,8 @@ protected void checkOptions() {
      * Parse options string and add all recognized options and their values.
      * If optionString is <i>null</i> this method just does nothing.
      *
-     * @throws BadOption if known option has illegel value
-     *                  or all options are inconsistent
+     * @throws BadOption if known option has illegal value
```

Comment:
Shouldn't BadOption use \<i\> or \<code\>?

Reply:
no, `@throws` should be followed by exception class-name, see [1]

[1]: https://docs.oracle.com/en/java/javase/15/docs/specs/javadoc/doc-comment-spec.html#throws

Reply:
ok